cmd/security-spiffe-token-provider/res/configuration.toml

```diff
@@ -38,6 +38,17 @@ Port = 6379
 Timeout = 5000
 Type = "redisdb"
 
+
+[TokenConfig]
+#PrivilegedTokenPath = "UNUSED"
+#ConfigFile = "UNUSED"
+#OutputDir = "UNUSED"
+#OutputFilename = "UNUSED"
```

Comment:
if we remove this, shouldn't it become breaking changes in terms of update pov?

Reply:
What this is actually doing is reusing the `configuration.toml` snippet from `security-file-token-provider` except that we don't need all of the parameters, just some of them.  I'm just making it clear that we don't use these parameters in `spiffe-file-token-provider`.

Reply:
ok